ppu/librt/utime.c

```diff
@@ -0,0 +1,47 @@
+#include <sys/file.h>
+#include <time.h>
+#include <utime.h>
+#include <sys/errno.h>
+
+extern int errno;
+
+int __attribute__((weak)) utime(const char *filename, const struct utimbuf *times)
+{
+	s32 ret;
+	if (times == NULL) {
+		struct utimbuf now_time;
+		time_t     now;
+		now = time(NULL);
+		now_time.actime = now;
+		now_time.modtime = now;
+		ret = sysLv2FsUtime(filename,(sysFSUtimbuf *)&now_time);
+	}
+	else ret = sysLv2FsUtime(filename,(sysFSUtimbuf *)times);
+	if (ret != SYS_FS_OK ) {
+		switch (ret) {
```

Comment:
The error conversion is already available here:
https://github.com/ps3dev/PSL1GHT/blob/master/ppu/librt/lv2errno.c

Reply:
Good!!
So i can remove utime.h